packages/core/src/external/carequality/ihe-gateway-v2/outbound/xca/process/error.ts

```diff
@@ -39,6 +39,14 @@ export function processRegistryErrorList(
 
       operationOutcome.issue.push(issue);
     });
+
+    const msg = "RegistryErrorList is present in response";
+    capture.error(msg, {
+      extra: {
+        outboundRequest,
+        registryErrorList,
+      },
```

Comment:
Thinking we could help ourselves by exposing the `requestId` as its own prop on `extra` so its easier to debug/dig through events when trying to diagnose IHE GW related stuff. I know its part of `outboundRequest`, but in each situation/part of the code it might be on a diff place w/ a diff name, so thinking if we can standardize that, at least for IHE GW related stuff.